packages/viz/src/lib/style/palettes/palette.ts

```diff
@@ -0,0 +1,81 @@
+import { SubPalette } from 'cartocolor';
+
+/**
+ * Color palettes.
```

Comment:
Nice reuse here!

Reply:
Sadly, I have not been able to found any. I'm gonna create another branch for testing the code.

Reply:
don't stop now for the PoC, as palette is quite tested (on vl)